### ISSUE TITLE
Support configuring different image registries

### ISSUE THREAD

**Issue by simu** [projectsyn/component-csi-cloudscale#20]
## Context

Currently, the component doesn't allow customizing the image registries for the container images used by the component. Being able to configure a different registry can be useful in cases where a registry mirror is required (e.g. to avoid dockerhub pull rate limits).

## Alternatives

Deal with intermittent pods in ImagePullBackOff due to pull rate limits.

**Comment by megian** [comment]
Relates to cloudscale-ch/csi-cloudscale#31.

**Comment by megian** [comment]
@simu Can we close this issue if favor of #53, where the registry can be overridden?

**Comment by simu** [comment]
not really, since the component best practices recommend having separate parameters for the Helm chart and container images (cf. https://syn.tools/syn/explanations/commodore-components/container-images.html and https://syn.tools/syn/explanations/commodore-components/helm-charts.html). To ensure we get a single PR which updates both the Helm chart and the container image, we can configure Renovate to group the updates, cf. https://github.com/projectsyn/component-rook-ceph/blob/1a5be5e0a4bbd72659c546ac39eb83d19a284474/renovate.json#L18-L23
